ts/client/scripts/risk.ts

```diff
@@ -185,18 +196,27 @@ async function computePriceImpactForLiqor(
     // liabs by the liqor, who would immediately want to reduce to 0
     // Assuming assets need to be sold to USDC
     const assets = mangoAccountsWithHealth.reduce((sum, a) => {
-      if (bank.maintAssetWeight.eq(ZERO_I80F48())) {
-        return sum.add(ZERO_I80F48());
-      }
       // How much would health increase for every unit liab moved to liqor
+      // assetprice * (liabweight/(1+liabliqfee) - assetweight)
       const tokenAssetHealthContrib = ONE_I80F48()
         .sub(bank.liquidationFee)
         .mul(bank.price)
-        .mul(bank.maintAssetWeight);
+        .mul(bank.initAssetWeight);
+      bank.price.mul(
+        Array.from(group.banksMapByTokenIndex.values())
+          .flat()
+          .map((bank) => bank.initLiabWeight)
+          .reduce((prev, curr) => (prev.lt(curr) ? prev : curr))
+          .div(ONE_I80F48().add(bank.liquidationFee))
+          .sub(bank.initAssetWeight),
+      );
```

Comment:
I don't understand this bit. Does it have a side effect? Shouldn't tokenAssetHealthContrib be different? (according to the comment you added?)

Reply:
wow ok, I had written the new version on a new line, and forgot to remove the old one, done